sycl/test-e2e/KernelAndProgram/build-log.cpp

```diff
@@ -1,5 +1,6 @@
 // for CUDA and HIP the failure happens at compile time, not during runtime
 // UNSUPPORTED: cuda || hip || ze_debug
+// TODO: rewrite this into a unit-test
```

Comment:
Since the whole intent of this PR is to minimize amount of changes in "drop PI" PR, I'm postponing this rewrite to reduce amount of conflicts.